src/ducks/transfers/TransferPage.jsx

```diff
@@ -79,15 +98,58 @@ const ChooseRecipientCategory = translate()(
   }
 )
 
+const BeneficiaryRow = ({ beneficiary, onSelect }) => {
+  return (
+    <Row
+      className="u-clickable"
+      onClick={onSelect.bind(null, beneficiary)}
+      key={beneficiary._id}
```

Comment:
Why do you use a `key` outside of a `map` ?

Reply:
Forgotten key, thanks